themes/nord.json

```diff
@@ -842,7 +842,7 @@
       "name": "[JavaScript] Variable Other Object",
       "scope": "source.js meta.var.expr variable.other.object",
```

Comment:
Also removing the `meta.var.expr` scope fixes the problem with inconsistent highlighting.